src/utils/sortTypes.ts

```diff
@@ -7,7 +7,7 @@ export const sortTypes = {
     date: (v1: any, v2: any, id: string) => {
         const a = new Date(v1?.values?.[id] || 0);
         const b = new Date(v2?.values?.[id] || 0);
-        return b?.getTime() - a?.getTime();
+        return a?.getTime() - b?.getTime();
```

Comment:
This seems to have been the opposite of what we wanted? At least the way I interpret it, that's why I made this change.